storage/src/vespa/storage/distributor/top_level_bucket_db_updater.cpp

```diff
@@ -124,8 +118,8 @@ TopLevelBucketDBUpdater::remove_superfluous_buckets(
     const char* up_states = storage_node_up_states();
     // TODO STRIPE explicit space -> config mapping, don't get via repo
     //   ... but we need to get the current cluster state per space..!
-    for (auto& elem : _op_ctx.bucket_space_repo()) {
-        const auto& old_cluster_state(elem.second->getClusterState());
+    for (auto& elem : _op_ctx.bucket_space_states()) {
```

Comment:
The above TODO can probably be removed now

Reply:
Yes, fixed.